demo/embedded_demo.html

```diff
@@ -0,0 +1,93 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>OpenSheetMusicDisplay as an embedded iframe</title>
+    <meta name="description" content="A showcase for OpenSheetMusicDisplay.">
+    <meta name="author" content="OpenSheetMusicDisplay contributors">
+
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js" type="text/javascript"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.3.0/semantic.min.js"
+            type="text/javascript"></script>
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.3.0/semantic.css" media="all" rel="stylesheet"/>
+    <link rel="icon" href="./favicon.ico?" type="image/x-icon"/>
+</head>
+<body>
+<br>
+<div style="text-align: center;width: 60%;margin:0 auto">
+    <h1 class="ui centered header" id="header">
+        <img src="./favicon.ico?" class="ui image">
+
+    </h1>
+    <h2>
+        OpenSheetMusicDisplay as an embedded iframe
+    </h2>
+</div>
+<div style="padding: 40px;margin:0 auto">
+    <h3>
+        Use parameters:
+    </h3>
+    <p>
+    <table class="ui table celled ">
+        <tr>
+            <th>Parameter</th>
+            <th>Values</th>
+            <th>Default</th>
+            <th>Description</th>
+        </tr>
+        <tr>
+            <td><b>embedded</b></td>
+            <td>"0" and "1"</td>
+            <td>"0" = OFF</td>
+            <td>Activates the embedded mode: controls are hidden (unless other params are given) and sheet will be
+                rendered more compactly
+            </td>
+        </tr>
+        <tr>
+            <td>showControls</td>
+            <td>"0" and "1"</td>
+            <td>"0" = OFF</td>
+            <td>If active, the controls of OSMD are shown like usually.</td>
```

Comment:
some controls like zoom level are shown, not all OSMD controls like options, re-render and so on.

Reply:
we need to define now what controls and other stuff we want to allow the embedder to customize/hide/show.
I would say only developers would need something like re-render which should not be part of the embedding options.

Reply:
But on this demo page should change the text "the controls of OSMD are shown like usually" to be more correct.
Also the "Default" values given don't seem to be correct, right? At least the showZoomControl seems to be on by default, but off is written there.